uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/daemon/ssh/GitSSHService.java

```diff
@@ -137,29 +187,101 @@ public void setup(final File certDir,
         });
         sshd.setPublickeyAuthenticator(new CachingPublicKeyAuthenticator((username, key, session) -> {
 
-            final FileSystemUser user = getSshAuthenticator().authenticate(username, key);
+            final FileSystemUser user = getSshAuthenticator().authenticate(username,
```

Comment:
avoid change format where you don't need

Reply:
Ok